src/ImageSharp/ImageFrameCollectionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp;
+
+/// <summary>
+/// Extension methods for <see cref="ImageFrameCollection{TPixel}"/>.
+/// </summary>
+public static class ImageFrameCollectionExtensions
```

Comment:
This is required because `ImageFrameCollection<TPixel>` implements both `IEnumerable<ImageFrame>` via inheritance and `IEnumerable<ImageFrame<TPixel>>`. You have to awkwardly cast to use `Select` and `Where` etc.

`Select` can easily be pollyfilled but an attempt to polyfill `Where` or other members of `System.Linq` will clash to it's easier to add a single `AsEnumerable<TPixel>` method which can then be chained against for the `System.Linq` methods.